Gopkg.toml

```diff
@@ -93,6 +103,10 @@
   name = "k8s.io/client-go"
   version = "kubernetes-1.9.3"
 
+[[constraint]]
+  branch = "release-1.9"
+  name = "k8s.io/kubernetes"
```

Comment:
I had to pin this. There was no other way I found to get the stupid deps right. This maybe improves again with the move to `1.10.4`.